src/main/java/com/ankist/expensetracker/controller/ExpenseRestService.java

```diff
@@ -0,0 +1,46 @@
+package com.ankist.expensetracker.controller;
+
+import com.ankist.expensetracker.model.ExpenseDTO;
+import com.ankist.expensetracker.repository.ExpenseDAO;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+public class ExpenseRestService {
+
+    @Autowired
+    private ExpenseDAO expenseServiceDAO;
+
+
+    @GetMapping("/getById")
+    public ResponseEntity<ExpenseDTO> findById(@RequestParam("id") Integer id){
+
+        return new ResponseEntity<>(expenseServiceDAO.getExpenseById(id),HttpStatus.OK);
+    }
+
+    @GetMapping("/getAll")
+    public ResponseEntity<List<ExpenseDTO>> findAll(){
+        return new ResponseEntity<>(expenseServiceDAO.getExpenseAll(),HttpStatus.OK);
+    }
+
+    @GetMapping("/deleteById")
```

Comment:
Should be @DeleteMapping without a path definition
```suggestion
    @DeleteMapping
```